src/colp/components/manual_correction_report.py

```diff
@@ -0,0 +1,42 @@
+import streamlit as st
+import pandas as pd
+
+
+class ManualCorrection:
+    def __init__(self, data):
+        self.modifications_applied = data["modifications_applied"]
+        self.modifications_not_applied = data["modifications_not_applied"]
+
+    def __call__(self):
+        st.subheader("Manual Correction Report")
+
+        if (
+            self.modifications_applied is None
+            and self.modifications_not_applied is None
+        ):
+            st.info("No Manual Correction Table Available.")
+            return
+        self.display_manual_correction()
+
+    def display_manual_correction(self):
+        applied = self.modifications_applied
+        not_applied = self.modifications_not_applied
+        if not applied.empty:
+            count1 = applied.groupby("field").size()
+            count1 = count1.reset_index()
+            count1.rename(
+                columns={"field": "Field", 0: "Number of Records"}, inplace=True
+            )
+            st.subheader("Numbers of Manual Corrections Applied By Field")
+            st.write(count1)
+        if not not_applied.empty:
+            count2 = not_applied.groupby("field").size()
+            count2 = count2.reset_index()
+            count2.rename(
```

Comment:
I think actually we should send reorder columns so `uid` is on the right. Easier way to implement this though, all you need is something like
```python3
count2 = count2[count2.columns[1:] + ["uid"]]
```

 this [this blog post](https://stackoverflow.com/questions/13148429/how-to-change-the-order-of-dataframe-columns) has more detail on how to do this